### ISSUE TITLE
failed  {"message":"Forbidden"} | AssertionError: expected 403 to be one of [ 200, 201, 202 ]<title>

### ISSUE THREAD

**Issue by tebissi** [a85/POSTMan-Chrome-Extension#10339]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

Having this error in postman 

 {"message":"Forbidden"} | AssertionError: expected 403 to be one of [ 200, 201, 202 ]

When running https://postman-echo.com/get

can someone please help

### Steps To Reproduce

 {"message":"Forbidden"} | AssertionError: expected 403 to be one of [ 200, 201, 202 ]

When running https://postman-echo.com/get

can someone please help

### Environment Information

```markdown
- Operating System: Windows 10
- Platform Type: Windows
- Postman Version: 9.05 version
```

**Comment by xk0der** [comment]
Hey @tebissi 👋 ! 

Looks like you may be using a Proxy or VPN while trying to access https://postman-echo.com/get
Make sure that access to `postman-echo.com` is not blocked by your Proxy/VPN or by Firewall
If you are using a corporate device, check with your IT Team to make sure access to `postman-echo.com` is not blocked.

**Try out the following for troubleshooting:**
* Try opening https://postman-echo.com/get in your browser and see if that works.
* Try disabling the proxy, switch off the VPN and see if things work after that.
* Also see if connecting to another network (for example your Phone's hotspot) makes any difference

**Comment by xk0der** [comment]
Hey @tebissi  I'm closing this issues, since we did not hear back from you. Feel free to log a new ticket if find any other bug.
